features/step_definitions.rb

```diff
@@ -108,6 +122,16 @@
 
 #
 
+Given(%r!^I have the following documents? under the "(.*)" collection within the "(.*)" directory:$!) do |label, dir, table|
+  table.hashes.each do |input_hash|
+    title = slug(input_hash["title"])
+    path = File.join(dir, "_#{label}", "#{title}.md")
```

Comment:
only because that's how our default site gets created from `jekyll new` .. and I think, an existing step also has something similar set up.. not sure..